docs/aws/README.md

```diff
@@ -3,31 +3,35 @@
 This guide will provide a complete Karpenter installation for AWS.
 These steps are opinionated and may need to be adapted for your use case.
 
+>Note: This guide should take less than 1 hour to complete and should cost less than $.25
```

Comment:
You can remove `Note` since it's already a >

Reply:
I'm assuming I should remove that for all of the `Note:`s